02week/ticTacToe.js

```diff
@@ -24,6 +24,48 @@ function printBoard() {
   console.log('2 ' + board[2].join(' | '));
 }
 
+//do we need to associate the board with this section? 
+
+//this is a helper function for checkForWin()
+//return true if any row wins
+function horizontalWin() {
+  // Your code here
+  //loops through each option for the second array
+  for (let i = 0; i >= 2; i++) {
+
+    //checked is any of the whole rows returns with the same value 
+    if (((board[0][i]) || (board[1][i]) || (board[2][i])) === playerTurn) 
```

Comment:
this does not check for all 3 cells being equals. you would have to check each pair separately
```
if(board[0][i] == playerTurn && board[1][i] == playerTurn && board[2][i] == playerTurn)

```